### DISCUSSION TITLE
Using maxValues with option groups

### DISCUSSION THREAD

**Discussion by jakubsob** [sa-si-dev/virtual-select#236] [Q&A] [answered]
Hello,

it is possible to use `maxValues` with option groups?

When setting `maxValues` to a number other than 0 group selection checkbox gets removed automatically. Is there a way to be able to select first `n` values when clicking on selection of a whole group?

### Case `maxValues == 0`
![image](https://user-images.githubusercontent.com/37193264/224675056-51f4985f-bdb7-42bc-be89-c8a7dc02f7a3.png)

### Case `maxValues == 5`
![image](https://user-images.githubusercontent.com/37193264/224675424-828222a2-e295-4117-afaf-4c3de307ac8d.png)

Thank you

**Comment by sa-si-dev** [comment, marked as answer]
@jakubsob as of now, select all / select group checkboxes would be hidden when maxValue is given.

**Reply by jakubsob** [reply]
Thank you 👍
